### PR TITLE
Basic task creation ui

### LINKED ISSUE #958: Add the ability to create a task
as a task builder
with editor permissions or higher
when i view the tasks page
then i have the ability to create a task through pasting a task text to the existing flux script editor component

### PR DESCRIPTION
**Pull request by bthesorceror**
Closes #958 

Basic UI for creating a task that belongs to the user

  - [x] Rebased/mergeable
  - [x] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: chronograf/ui/src/flux/components/v2/FluxEditor.test.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  const wrapper = shallow(<FluxEditor {...props} />)
+
+  return {
+    wrapper,
+    props,
+  }
+}
+
+describe('Flux.Components.FluxEditor', () => {
```
Comment (OfTheDelmer) [inline chronograf/ui/src/flux/components/v2/FluxEditor.test.tsx]: Nice

File: chronograf/ui/src/index.tsx [hunk 2]
```diff
@@ -109,6 +111,8 @@ class Root extends PureComponent<{}, State> {
                     path="dashboards/:dashboardID"
                     component={DashboardPage}
                   />
+                  <Route path="tasks" component={TasksPage} />
```
Comment (OfTheDelmer) [inline chronograf/ui/src/index.tsx]: :)

File: chronograf/ui/src/shared/apis/v2/organization.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import _ from 'lodash'
+import AJAX from 'src/utils/ajax'
+
+interface User {
```
Comment (OfTheDelmer) [inline chronograf/ui/src/shared/apis/v2/organization.ts]: Did we want to separate this into a types/v2 file somewhere. Seems like it's used here and in user.

File: chronograf/ui/src/shared/apis/v2/organization.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  name: string
+  id: string
+}
+
+export const getOrganizations = async (url: string): Promise<User> => {
+  const {data} = await AJAX({
+    url,
+  })
+
+  return _.get(data, 'orgs', [])
```
Comment (OfTheDelmer) [inline chronograf/ui/src/shared/apis/v2/organization.ts]: Do we prefer `getDeep<User>`? I see it and `_.get` so I'm not too sure which we prefer.

File: chronograf/ui/src/shared/apis/v2/organization.ts (hunk 3 above)
Comment (OfTheDelmer) [reply chronograf/ui/src/shared/apis/v2/organization.ts]: Or we could call this one `Organization` so it returns `Promise<Organization>`

File: chronograf/ui/src/shared/copy/v2/notifications.ts [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import {Notification} from 'src/types'
+import {TEN_SECONDS} from 'src/shared/constants/index'
+
+type NotificationExcludingMessage = Pick<
```
Comment (OfTheDelmer) [inline chronograf/ui/src/shared/copy/v2/notifications.ts]: :)

File: chronograf/ui/src/tasks/api/v2/index.ts [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import AJAX from 'src/utils/ajax'
+export const submitNewTask = (url, owner, org, flux) => {
```
Comment (OfTheDelmer) [inline chronograf/ui/src/tasks/api/v2/index.ts]: May need some breathing  after imports

File: chronograf/ui/src/tasks/containers/TaskPage.tsx [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+import {Links} from 'src/types/v2/links'
+import {State as TasksState} from 'src/tasks/reducers/v2'
+import {setNewScript, saveNewScript} from 'src/tasks/actions/v2'
+import {
+  Form,
+  Columns,
+  ComponentColor,
+  ComponentSize,
+  Button,
+} from 'src/clockface'
```
Comment (OfTheDelmer) [inline chronograf/ui/src/tasks/containers/TaskPage.tsx]: Nice

File: chronograf/ui/src/tasks/containers/TaskPage.tsx [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+import {Page} from 'src/pageLayout'
+
+interface ConnectStateProps {
```
Comment (OfTheDelmer) [inline chronograf/ui/src/tasks/containers/TaskPage.tsx]: I like the Connect prefix

File: chronograf/ui/src/tasks/containers/TasksPage.tsx [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+            <Button
+              color={ComponentColor.Primary}
+              onClick={this.handleCreateTask}
+              icon={IconFont.Plus}
+              text="Create Task"
+              titleText="Create a new task"
+            />
+          </Page.Header.Right>
+        </Page.Header>
+        YO!
```
Comment (OfTheDelmer) [inline chronograf/ui/src/tasks/containers/TasksPage.tsx]: haha

**Comment by OfTheDelmer** [comment]
LGTM, left some nits

File: chronograf/ui/src/shared/apis/v2/organization.ts (hunk 4 above)
Comment (bthesorceror) [reply chronograf/ui/src/shared/apis/v2/organization.ts]: one level deep `_.get` works fine

File: chronograf/ui/src/shared/apis/v2/organization.ts (hunk 3 above)
Comment (bthesorceror) [reply chronograf/ui/src/shared/apis/v2/organization.ts]: good catch

**Pull request merged**